src/Android/Xamarin.Android/Samples/Augmented Reality/ExploreScenesInFlyoverAR/ExploreScenesInFlyoverAR.cs

```diff
@@ -0,0 +1,114 @@
+using Esri.ArcGISRuntime.UI;
+using System;
+
+namespace ArcGISRuntimeXamarin.Samples.ExploreScenesInFlyoverAR
+{
+    [Activity(ConfigurationChanges = Android.Content.PM.ConfigChanges.Orientation | Android.Content.PM.ConfigChanges.ScreenSize)]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Explore scenes in flyover AR",
+        "Augmented reality",
+        "Use augmented reality (AR) to quickly explore a scene more naturally than you could with a touch or mouse interface.",
+        "")]
+    public class ExploreScenesInFlyoverAR : AppCompatActivity
+    {
+        // Hold references to the UI controls.
+        private ARSceneView _arSceneView;
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Explore scenes in flyover AR";
+
+            CreateLayout();
+        }
+
+        private void CreateLayout()
+        {
+            // Create the layout.
+            LinearLayout layout = new LinearLayout(this) { Orientation = Orientation.Vertical };
+
+            // Add the AR scene view.
+            _arSceneView = new ARSceneView(this);
+            layout.AddView(_arSceneView);
+
+            SetContentView(layout);
+
+            // Request camera permission. Initialize will be called when permissions are granted.
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create the scene with a basemap.
+            Scene flyoverScene = new Scene(Basemap.CreateImagery());
+
+            // Create the integrated mesh layer and add it to the scene.
+            IntegratedMeshLayer meshLayer = new IntegratedMeshLayer(new Uri("https://www.arcgis.com/home/item.html?id=dbc72b3ebb024c848d89a42fe6387a1b"));
+            flyoverScene.OperationalLayers.Add(meshLayer);
+
+            try
+            {
+                // Wait for the layer to load so that extent is available.
+                await meshLayer.LoadAsync();
+
+                // Start with the camera at the center of the mesh layer.
+                Envelope layerExtent = meshLayer.FullExtent;
+                Camera originCamera = new Camera(layerExtent.GetCenter().Y, layerExtent.GetCenter().X, 600, 0, 90, 0);
+                _arSceneView.OriginCamera = originCamera;
+
+                // Set the translation factor to enable rapid movement through the scene.
+                _arSceneView.TranslationFactor = 1000;
+
+                // Enable atmosphere and space effects for a more immersive experience.
+                _arSceneView.SpaceEffect = SpaceEffect.Stars;
+                _arSceneView.AtmosphereEffect = AtmosphereEffect.Realistic;
+
+                // Display the scene.
+                await flyoverScene.LoadAsync();
+                _arSceneView.Scene = flyoverScene;
+            }
+            catch (Exception ex)
+            {
+                ShowMessage("Failed to start AR", "Error starting");
```

Comment:
You can replace the ShowMessage method with
`new AlertDialog.Builder(this).SetMessage("Failed to start AR.").SetTitle("Error").Show();`